core-bundle/src/Resources/contao/library/Contao/Database/Result.php

```diff
@@ -363,7 +366,13 @@ public function count()
 	{
 		if ($this->rowCount === null)
 		{
-			$this->rowCount = $this->resResult->rowCount();
+			if (
+				$this->resResult instanceof DoctrineStatement
+				|| $this->resResult instanceof DoctrineResult
+				|| $this->resResult instanceof DoctrineDriverResult
```

Comment:
`Doctrine\DBAL\Driver\Result` does not extend `Doctrine\DBAL\Driver\ResultStatement`, so based on the if statement in line 85, `$this->resResult instanceof DoctrineDriverResult ` can never be `true`, can it?